### PR TITLE
fix calcule of tiled texture

### PR DESCRIPTION
**Pull request by Zangetsu38**
- fix crash on one game with fix also render on it.
origin of pr for this code
https://github.com/Vita3K/Vita3K/pull/543 in commit https://github.com/Vita3K/Vita3K/commit/963c1699654a2ce694ec1f302d1a403ea0ee8531#diff-0a445e58d6c273c10266b1ed8e609cccbcaf2164c74780845336ee5720514e68
never touched again after

# Result:
fix cvrash on this game and get pêrfect ingame render
image
image

### CODE REVIEW

File: vita3k/renderer/src/texture_format.cpp [hunk 1]
```diff
@@ -180,10 +180,10 @@ void tiled_texture_to_linear_texture(uint8_t *dest, const uint8_t *src, uint16_t
     for (uint16_t y = 0; y < height; y++) {
         for (uint16_t x = 0; x < width; x++) {
             // Calculate texel address in title
-            const uint16_t texel_offset_in_title = (x & 0b11111) | ((y & 0b11111) << 4);
+            const uint16_t texel_offset_in_title = (x & 0b11111) | ((y & 0b11111) << 5);
             const uint16_t tile_address = (x & 0b111111100000) + width_in_tiles * (y & 0b111111100000);
 
-            uint32_t offset = ((tile_address << 9) | (texel_offset_in_title)) * (bits_per_pixel >> 3);
+            uint32_t offset = ((tile_address << 5) | (texel_offset_in_title)) * (bits_per_pixel >> 3);
```
Comment (Zangetsu38) [inline vita3k/renderer/src/texture_format.cpp:186]: this value is for fix crash, using 9/8 crash, using 7/6 get broken render with screen splitted

File: vita3k/renderer/src/texture_format.cpp [hunk 2]
```diff
@@ -180,10 +180,10 @@ void tiled_texture_to_linear_texture(uint8_t *dest, const uint8_t *src, uint16_t
     for (uint16_t y = 0; y < height; y++) {
         for (uint16_t x = 0; x < width; x++) {
             // Calculate texel address in title
-            const uint16_t texel_offset_in_title = (x & 0b11111) | ((y & 0b11111) << 4);
+            const uint16_t texel_offset_in_title = (x & 0b11111) | ((y & 0b11111) << 5);
```
Comment (Zangetsu38) [inline vita3k/renderer/src/texture_format.cpp:183]: here value is for can fix texture broken

**Comment by Zangetsu38** [comment]
also fix white album 2 with get ingame, in master it crash on call this function for can draw this logo
image
probleme here is logo is fine, but font is wrong probably get normally full white of font

**Comment by Zangetsu38** [comment]
also fix this game, but have strange glitch on main menu, only see here
image
crash after on create new character get by dynarmic

**Comment by Zangetsu38** [comment]
fix also crash on other toukiden game, but no sur if result is fine here
image

**Review by pent0 — APPROVED** [review]

**Pull request merged**